gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/internal/model/MetadataMongo.java

```diff
@@ -41,5 +43,6 @@ public class MetadataMongo extends Auditable {
 
     private String format;
 
+    @ExplicitEncrypted(algorithm = AEAD_AES_256_CBC_HMAC_SHA_512_Deterministic)
     private String value;
```

Comment:
Unrelated to encryption but after testing the metadata encryption I've noticed the updated date is never updated (there is a field mapping missing in the `MongoMetadataRepository.update()` method.

Reply:
The encryption of the metadata value works well by the way 🙂